src/doc/src/reference/features.md

```diff
@@ -513,3 +513,9 @@ source and inspect it.
 
 [`cargo vendor`]: ../commands/cargo-vendor.md
 [cargo-clone-crate]: https://crates.io/crates/cargo-clone-crate
+
+### Caveats
```

Comment:
I might give this section a more descriptive title, perhaps something like "Testing features"

Reply:
@ehuss sure, I think "testing" doesn't cover linting and the other examples though. WDYT?

Reply:
Yea, good point.  Maybe something like "Validating features" or "Feature combinations"?  Generally just to have a section title that isn't too generic.

Reply:
"Feature combinations" sounds great!